lib/util.js

```diff
@@ -142,6 +143,36 @@ api.parseLinkHeader = header => {
   return rval;
 };
 
+/**
+ * Parses a content-type header.
+ * The results will be key'd by the value of "rel".
+ *
+ * Accept: application/ld+json
+ *
+ * Parses as: ["application/ld+json", {}]
+ *
+ * Accept: application/ld+json;profile=http://www.w3.org/ns/json-ld#context
+ *
+ * Parses as: ["application/ld+json",
+ *             {profile: "http://www.w3.org/ns/json-ld#context"}]
+ *
+ * If there is more than one
+ *
+ * @param header the content-type header to parse.
+ */
+api.parseContentTypeHeader = header => {
```

Comment:
- Docs above are outdated, mentioning Accept and rel.
- Perhaps this should just use https://github.com/jshttp/content-type. It's a bit bigger but far more battle tested than a custom solution.

Reply:
Okay, updated to use that package.